docs/source/using_the_app.rst

```diff
@@ -77,7 +77,7 @@ Click on the links below to find more information about specific tabs.
    tabs/label_frames
    tabs/train_infer
    tabs/train_status
-   tabs/labeled_diagnostics
+   tabs/image_diagnostics
```

Comment:
leave as `tab/labeled_diagnostics`, this refers to the file name. if we update to `tab/image_diagnostics` then we'd need to change the file name as well (not necessary at the moment)